README.md

```diff
@@ -199,11 +199,13 @@ Prior to your first deployment, you'll need to do a few things:
 
 - Create a persistent volume for the sqlite database for both your staging and
   production environments. Run the following (feel free to change the GB size
-  based on your needs):
+  based on your needs and the region of your choice
+  (`https://fly.io/docs/reference/regions/`). NB. If you do change the region,
```

Comment:
I don't know what "NB" means here, but I think it makes sense without it.

```suggestion
  (`https://fly.io/docs/reference/regions/`). If you do change the region,
```

Reply:
Yep, works fine without. N.B. was "Abbreviation of Latin nota bene. Used before a piece of important information to make readers notice it."